redis/unlock.lua

```diff
@@ -28,5 +28,7 @@ else
   redis.call('DEL', unique_digest)  -- TODO: Legacy support (Remove in v6.1)
 end
 
-return redis.call('LPUSH', available_key, token)
+local count = redis.call('LPUSH', available_key, token)
+redis.call('EXPIRE', available_key, 5)
```

Comment:
Curious as to why you picked 5 seconds here. Worth making a constant?

Reply:
No particular reason, do you think 5 is too high? Too low? I mean the key doesn't really do anything except signal to locks that are waiting that the key is now available.

Reply:
Nah was just curious - sort of a magic number if you're reading that code without context